File: Fragments/app/src/main/java/com/example/lambda_school_loaner_47/fragments/PhoneActivity.java
package com.example.lambda_school_loaner_47.fragments;

import android.net.Uri;
import android.support.v4.app.Fragment;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;

public class PhoneActivity extends AppCompatActivity implements DetailsFragment.OnFragmentInteractionListener {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_phone);

        DetailsFragment fragment = DetailsFragment.newInstance((Person) getIntent().getSerializableExtra(MainActivity.PERSON_ITEM));
        getSupportFragmentManager().beginTransaction().replace(R.id.frameLayout, fragment).commit();
    }

    @Override
    public void onFragmentInteraction(Uri uri) {

    }
}

File: app/src/main/java/com/lambdaschool/congressfragmentsproject/PhoneActivity.kt
package com.lambdaschool.congressfragmentsproject

import android.net.Uri
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle

class PhoneActivity : AppCompatActivity(), DetailsFragment.OnFragmentInteractionListener{

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_phone)
    }

    override fun onFragmentInteraction(uri: Uri) {
        val fragment = DetailsFragment()
        val bundle = Bundle()
        val item_id = intent.getStringExtra("item_key")
        bundle.putString("item_key", item_id)

        supportFragmentManager.beginTransaction()
            .replace(R.id.phone_frame_layout, fragment)
            .commit()
    }
}
